### ISSUE TITLE
[Content]: CLI docs missing sentence

### ISSUE THREAD

**Issue by mikewink** [craftcms/docs#608]
### Product

Craft CMS

### Description

In the CLI section of the documentation, there is a paragraph that stops mid-sentence. 

`If you installed Craft using our [quick-start guide](../install.md), you’ve already [?] The Craft CLI executable typically …`.

**Comment by jasonlav** [comment]
I am seeing a similar issue, but slightly different.

```
If you installed Craft using our [quick-start guide](../install.md), you’ve already
```

https://github.com/craftcms/docs/edit/main/docs/5.x/system/cli.md Line 11

**Comment by AugustMiller** [comment]
Fixed in 716e5e531f4ba2475ef86cf1ff930e706e0f3612. Thanks for reporting!
